CHANGELOG.md

```diff
@@ -12,8 +12,11 @@ This project adheres to [Semantic Versioning](http://semver.org/).
   ([#921](https://github.com/nix-rust/nix/pull/921))
 - Added support for `SCM_CREDENTIALS`, allowing to send process credentials over Unix sockets.
   ([#923](https://github.com/nix-rust/nix/pull/923))
+- Added `kmod` module that allows loading and unloading kernel modules on Linux.
+  ([#930](https://github.com/nix-rust/nix/pull/930))
 
 ### Changed
+- Bump minimum tested Rust version to 1.22.1.
```

Comment:
"tested" -> "required"

Reply:
This commit/change will be removed once #900 is merged